tools/api-tests/src/xpub-api/submission.ts

```diff
@@ -255,9 +256,21 @@ const fileUploadProgress = async (_ctx: ApiTestContext): Promise<Manuscript> =>
   throw new NotImplementedError();
 };
 
+// Submits a survey
+const submitSurvey = async (ctx: ApiTestContext, data: SurveySubmission): Promise<{ submitSurvey: boolean}> => {
+  const query = `
+  mutation SubmitSurvey($data: SurveySubmission!) {
+  submitSurvey(data: $data)
+}
+  `;
+
+  return await withAuthorization(ctx, query, {data} );
+}
+
 const resolvers = {
   Query: { manuscript, editors, getSubmission},
   Mutation: {
+    submitSurvey,
```

Comment:
Maybe it should be `submitSurveyResponse` to match how we do it in #2292 ?

Reply:
Will change once #2304 is in, otherwise this will fail CI